docs/news/index.qmd

```diff
@@ -9,6 +9,33 @@ format:
 
 This page provides the release notes associated with each release of RStudio and Posit Workbench. Please contact customer support (<a href="mailto:support@posit.co">support@posit.co</a>) for questions about the described changes.
 
+## RStudio 2023.12.1
+
+**"Ocean Storm"**
+
+>Date: 2024-01-29
+
+### New
+#### RStudio
+- Changed the failure message for server no-responses (http code 0) to be more helpful to the user (rstudio/rstudio-pro#5646)
+
+
+#### Posit Workbench
+
+### Fixed
+#### RStudio
+- Removed GitHub Copilot preview label and disclaimer in Copilot options panes (#14067)
+- Fixed an issue where GitHub Copilot indexed binary files when project indexing enabled (#14106)
+- Fixed regression that prevented opening help topics in separate window (#14097)
+- Increased open files limit (rstduio/rstudio#14148)
+
+#### Posit Workbench
+- Fixed join session when ready control not responding to mouse clicks in all areas. (rstudio/rstudio-pro#5609)
```

Comment:
It doesn't look like we were using ending punctuation in the other entries so we can drop the period for consistency

```suggestion
- Fixed join session when ready control not responding to mouse clicks in all areas (rstudio/rstudio-pro#5609)
```

Reply:
Committed!